### PR TITLE
Add Double Commander Qt.app Latest

### PR DESCRIPTION
**Pull request by rajivnarayan**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Why have both versions?

File: Casks/double-commander-qt.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version '0.9.7-9235'
+  sha256 '9eb09d7d407ee88fae828966c1a460bce508db97e97ebea0559fdd20b9380a4d'
+
+  # downloads.sourceforge.net/doublecmd was verified as official when first introduced to the cask
+  url "https://downloads.sourceforge.net/doublecmd/doublecmd-#{version}.qt.x86_64.dmg"
+  appcast 'https://sourceforge.net/projects/doublecmd/rss'
+  name 'Double Commander Qt'
+  homepage 'https://doublecmd.sourceforge.io/'
+
+  app 'Double Commander.app', target: 'Double Commander Qt.app'
```
Comment (vitorgalvao) [inline Casks/double-commander-qt.rb]: [Renames require a reason](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/app.md#target-should-only-be-used-in-select-cases).

**Comment by vitorgalvao** [comment]
https://github.com/Homebrew/homebrew-cask/pull/76553 seems like a better solution.

**Comment by rajivnarayan** [comment]
ok, I prefer the Qt version anyway. The rename was to allow both versions to coexist.

Not sure why the two versions exist, but there are UI bugs that affect one but not the other. [This](https://doublecmd.sourceforge.io/mantisbt/view.php?id=2352) one in particular makes search using the latest cocoa version almost unusable.

File: Casks/double-commander-qt.rb (hunk 1 above)
Comment (rajivnarayan) [reply Casks/double-commander-qt.rb]: The rename was to allow both the cocoa and qt versions to coexist.

Comment (vitorgalvao) [reply Casks/double-commander-qt.rb]: I understand your reason, what I meant is that reason needs to be in the cask, and use one of the example comments.

Comment (rajivnarayan) [reply Casks/double-commander-qt.rb]: ok added a comment to the cask in my [branch](https://github.com/rajivnarayan/homebrew-cask/commit/78730786e5616c185288da41766d7d2b3ced5d2c), should I create a new PR?

Comment (vitorgalvao) [reply Casks/double-commander-qt.rb]: No, the main cask is already the QT version, there would be no point.

**Pull request closed without merging**